packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -105,6 +106,24 @@ export const BookingFields = ({
           );
         }
 
+        if (field?.options) {
+          const moreThanOneInPersonOptions = !!(
+            field.options.filter((field) => {
+              return field.value === DefaultEventLocationTypeEnum.InPerson;
+            }).length > 1
+          );
+
+          field.options = field.options.map((field) => {
+            return {
+              ...field,
+              value:
+                field.value === DefaultEventLocationTypeEnum.InPerson && moreThanOneInPersonOptions
```

Comment:
I think I saw similar logic elsewhere but trying to understand.
1. Why can't we use the` field.label ` approach in all cases? 
2. Wouldn't the same applicable to other `organizerInput` type locations and not just InPerson

Reply:
yes we should check for other organizer input type also. forgot to update this